### PR TITLE
Fix 5138: Replace some packages with native functions

### LINKED ISSUE #5138: Replace some packages with native functions
**Is your enhancement request related to a problem? Please describe.**
Some of the packages used can be easily replaced by native JS functions. This will reduce the overall size.

**Describe the solution you'd like**
Replace 'copy-text-to-clipboard', 'country-code-emoji' and 'query-string' packages with native functions.

### PR DESCRIPTION
**Pull request by KubesDavid**
# Pull Request Template

## Description

Replace 'copy-text-to-clipboard', 'country-code-emoji' and 'query-string' packages with native functions.

Fixes #5138

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/javascript/dashboard/helper/flag.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Gets emoji flag for given locale.
+ *
+ * @param {string} countryCode locale code
+ * @return {string} emoji flag
+ *
+ * @example
+ * getCountryFlag('cz') // '🇨🇿'
+ */
+export const getCountryFlag = countryCode => {
```
Comment (pranavrajs) [inline app/javascript/dashboard/helper/flag.js:12]: Screenshot 2022-07-28 at 8 06 46 PM


Let us revert this change. The country-code-emoji does the exact same thing. I don't there is a significant advantage of us managing the code.

**Comment by pranavrajs** [comment]
@KubesDavid Thanks for the suggestion and the PR. I think the changes in the PR don't make much difference in the bundle size as the packages (clipboard, country-code) are <0.5kb in size and (query-string) is only 2kb. 

Let me know if there is any other advantage you can see here.

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfoRow.vue [hunk 2]
```diff
@@ -62,8 +61,9 @@ export default {
   methods: {
     onCopy(e) {
       e.preventDefault();
-      copy(this.value);
-      this.showAlert(this.$t('CONTACT_PANEL.COPY_SUCCESSFUL'));
+      navigator.clipboard.writeText(this.value).then(() => {
+        this.showAlert(this.$t('CONTACT_PANEL.COPY_SUCCESSFUL'));
+      });
```
[the remark this answers is not in the document]
Comment (KubesDavid) [reply app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfoRow.vue]: Good idea! Done.

**Comment by KubesDavid** [comment]
My inspiration behind this PR is mainly the fact that only one feature is used from these packages, which is easy to replace. There is no need to add unnecessary packages to every single function, it avoids the mess in packages.json.

**Comment by pranavrajs** [comment]
@iamsivin @fayazara Please test changes and see if we are good to merge this PR.

File: app/javascript/shared/helpers/clipboard.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Writes a text string to the system clipboard.
+ *
+ * @param {string} text text to be written to the clipboard
+ * @param {function} successCallback callback on successful copy
+ * @param {function} errorCallback callback on error copy
+ */
+export const copyTextToClipboard = (
```
Comment (fayazara) [inline app/javascript/shared/helpers/clipboard.js:8]: We can use ES6 standards here. 
```js
export const copyTextToClipboard = async (text) => {
    try {
        await navigator.clipboard.writeText(text);
    } catch (error) {
        throw `Unable to copy text to clipboard. Err: ${error}`;
    }
}
```

**Review by fayazara — CHANGES_REQUESTED** [review]
Left a comment. Please have a look.

**Review by iamsivin — COMMENTED** [review]
Please see the comments

**Review by iamsivin — APPROVED** [review]
LGTM

**Pull request merged**